app/controllers/group-chats.js

```diff
@@ -0,0 +1,75 @@
+const messageModel = require('../models/message');
+
+const messages = {
+
+    getOneMessage: (req, res) => {
+        let userID = req.body.user.sourceID;
+
+        messageModel.findBySourceID(userID, (error, results) => {
+            if (error) {
+                throw error;
+            } else {
+                res.status(200).send(results);
+            }
+        });
+    },
```

Comment:
Added a function for getting one message basing on the sender's ID. The only part needed in the request is the user's ID.